web/sites/default/config/clamav.settings.yml

```diff
@@ -0,0 +1,15 @@
+enabled: true
```

Comment:
This doesn't look destructive, but it is already set in the prod config. I'm not sure why it was included.